src/bundle/Resources/views/themes/admin/system_info/my_ibexa.html.twig

```diff
@@ -54,3 +46,48 @@
         </table>
     </div>
 </section>
+
+<section class="ez-fieldgroup">
+    <h2 class="ez-fieldgroup__name">{{ 'ibexa.subscription'|trans|desc('Subscription') }}</h2>
+    <div class="ez-fieldgroup__content">
+        <table class="table ez-table ez-table--list ez-table--to-left">
+            <tbody>
+            {% if info.subscriptionExpiryDate is not empty %}
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.is_trial'|trans|desc('Is Trial') }}</td>
+                    <td class="ez-table__cell">
+                        <svg class="ez-icon ez-icon--small">
+                            <use xlink:href="{{ ez_icon_path(info.isTrial ? 'checkmark' : 'discard') }}"></use>
+                        </svg>
+                    </td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.subscription_expiry'|trans|desc('Subscription renewal date') }}*</td>
+                    <td class="ez-table__cell">
+                        {{ info.subscriptionExpiryDate|date('F Y') }}
+                    </td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td colspan="2">
+                        <hr/>
+                        * {{ 'ibexa.subscription_read_more'|trans|desc('Subscription renewal date does not reflect auto renawal, and typically does not represent when the subscription ends') }}
```

Comment:
```suggestion
                        * {{ 'ibexa.subscription_read_more'|trans|desc('Subscription renewal date does not reflect auto renewal, and typically does not represent when the subscription ends') }}
```
Side: I really have no idea what this sentence means, so unless you can assume the users will, this should be clarified.

Reply:
In short we can maybe skip this, as I called the data renewal date instead of "subscription expiry".


Long: Support expiry date from support.ibexa.co might not actually mean when  it will expire as contract might be agreed to be "auto renewal", so it will be updates a few weeks before expiring by script @mateuszbieniek / @glye  on support site.

Reply:
Skipped for now